main.tf

```diff
@@ -343,3 +343,10 @@ resource "time_sleep" "wait_on_destroy" {
   # Sleep for 5 minutes to allow Karpenter to clean up resources
   destroy_duration = "5m"
 }
+
+# Store cluster name info for app deployment
+resource "aws_ssm_parameter" "cluster_name" {
+  name  = "/platform/${local.stack_name}/name"
```

Comment:
Could you change it to cluster_name since that is the output. Also, I think it would be better to use the module output for the value since we might at one point for what ever reason change the logic of what stack_name means but cluster_name will always be the cluster_name output of the module.